packages/console-client/src/client/http/axios/errors.ts

```diff
@@ -0,0 +1,12 @@
+const InvaidArgumentExceptions = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  ThrowIfFalsy: (propName: string, prop: any) => {
```

Comment:
this is an "AssertTruthy" method, i'd name it this way

Reply:
done